src/App.tsx

```diff
@@ -1,7 +1,49 @@
 import React from 'react';
 
+type State = {
+  pressedKey: boolean;
+  pressedUserKey: string;
+};
+
+export class Click extends React.Component<{}, State> {
```

Comment:
Better create an interface for props even if it's an empty object (it'll be easier to update on future)